vips/vip-417/bscmainnet.ts

```diff
@@ -0,0 +1,204 @@
+        params: [
+          bscmainnet.NORMAL_TIMELOCK,
+          LzChainId.ethereum,
+          ethers.utils.defaultAbiCoder.encode(["address"], [ETHEREUM_VTREASURY]),
+          ETHEREUM_TOTAL_AMOUNT,
+          [bscmainnet.NORMAL_TIMELOCK, ethers.constants.AddressZero, ADAPTER_PARAMS],
+        ],
+        value: BRIDGE_FEES.toString(),
+      },
+      {
+        target: ETHEREUM_VTREASURY,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [ETHEREUM_XVS, ETHEREUM_TOTAL_AMOUNT, ethereum.NORMAL_TIMELOCK],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: XVS_BRIDGE,
+        signature: "sendFrom(address,uint16,bytes32,uint256,(address,address,bytes))",
+        params: [
+          bscmainnet.NORMAL_TIMELOCK,
+          LzChainId.arbitrumone,
+          ethers.utils.defaultAbiCoder.encode(["address"], [ARBITRUM_ONE_VTREASURY]),
+          ARBITRUM_ONE_TOTAL_AMOUNT,
+          [bscmainnet.NORMAL_TIMELOCK, ethers.constants.AddressZero, ADAPTER_PARAMS],
+        ],
+        value: BRIDGE_FEES.toString(),
+      },
+      {
+        target: ARBITRUM_ONE_VTREASURY,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [ARBITRUM_ONE_XVS, ARBITRUM_ONE_TOTAL_AMOUNT, ethereum.NORMAL_TIMELOCK],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: XVS_BRIDGE,
+        signature: "sendFrom(address,uint16,bytes32,uint256,(address,address,bytes))",
+        params: [
+          bscmainnet.NORMAL_TIMELOCK,
+          LzChainId.zksyncmainnet,
+          ethers.utils.defaultAbiCoder.encode(["address"], [ZKSYNCMAINNET_VTREASURY]),
+          ZKSYNCMAINNET_TOTAL_AMOUNT,
+          [bscmainnet.NORMAL_TIMELOCK, ethers.constants.AddressZero, ADAPTER_PARAMS],
+        ],
+        value: BRIDGE_FEES.toString(),
+      },
+      {
+        target: ZKSYNCMAINNET_VTREASURY,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [ZKSYNCMAINNET_XVS, ZKSYNCMAINNET_TOTAL_AMOUNT, ethereum.NORMAL_TIMELOCK],
+        dstChainId: LzChainId.zksyncmainnet,
+      },
+      ...ETHEREUM_REWARD_DISTRIBUTION_TARGETS.map(({ target, dstChainId, amount, treasury }) => ({
+        target: treasury,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [ETHEREUM_XVS, amount, target],
+        dstChainId,
+      })),
+      ...ARBITRUM_ONE_REWARD_DISTRIBUTION_TARGETS.map(({ target, dstChainId, amount, treasury }) => ({
+        target: treasury,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [ARBITRUM_ONE_XVS, amount, target],
+        dstChainId,
+      })),
+      ...ZKSYNCMAINNET_REWARD_DISTRIBUTION_TARGETS.map(({ target, dstChainId, amount, treasury }) => ({
+        target: treasury,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [ZKSYNCMAINNET_XVS, amount, target],
+        dstChainId,
+      })),
```

Comment:
The funds are not in the treasury anymore, right? Because of the commands `withdrawTreasuryToken` on the destination network. With those funds, we are withdrawing the XVS from the Treasury to the Timelock. I think we could remove those commands:

* lines 138-143 for Ethereum
* lines 156-161 for Arbitrum
* lines 174-179 for ZKSync